### DISCUSSION TITLE
Custom benchmark

### DISCUSSION THREAD

**Discussion by taveltjuven** [OpenBB-finance/OpenBB#2163] [Q&A] [answered]
Hi Guys! In porfolio, I am trying to use another benchmark other than the ETFs listed under the "bench" function (SPY, IVV, VTI etc). In my case i would like use a Swedish ETF called XACTOMXS30. How can i add another ETF to the benchmark list? I have been trying to find where in the code the pre configured benchmark list resides but comming up short. All help very appreciated! Cheers!

**Comment by DidierRLopes** [comment]
@JerBouma can you help here please?

**Comment by JerBouma** [comment, marked as answer]
Hi @taveltjuven,

The ETFs listed under the `bench` functionality are not the only ones you can choose. We curated this list because those are the ones that are frequently used as a benchmark.

In your case, I would search Yahoo Finance for your product. See below:
<img width="1058" alt="image" src="https://user-images.githubusercontent.com/46355364/181462863-c137a95d-649b-4d89-b00f-6bb325bbeadd.png">

Then, you could do the following:
<img width="672" alt="image" src="https://user-images.githubusercontent.com/46355364/181463674-171041f6-7881-4978-ae7d-20bab697ec85.png">

Unfortunately Yahoo Finance does not have any historical data on this ETF but you can use this as an example that you can enter any other ETF that isn't in our list.

**Reply by taveltjuven** [reply]
Awesome, thanks a lot! Very appreciated.
